packages/commonwealth/server/routes/updateChain.ts

```diff
@@ -202,6 +204,12 @@ const updateChain = async (
   if (discord_bot_webhooks_enabled) {
     chain.discord_bot_webhooks_enabled = discord_bot_webhooks_enabled;
   }
+  if (directory_page_enabled) {
+    chain.directory_page_enabled = directory_page_enabled;
+  }
```

Comment:
If this is a boolean, then we'd want to allow true or false, so the check should be `typeof directory_page_enabled !== 'undefined'`, unless we want to prevent the value from being disabled

Reply:
I started hooking this up and figured out that this does not work. The thing is that `updateChain.ts` is not used anymore after merging https://github.com/hicommonwealth/commonwealth/pull/5199 . Now, the logic should land in `update_community_handler.ts` in `__updateCommunity` function. I think you guys should remove `updateChain` if it is not used but I will default it to @rbennettcw